aws/resource_aws_emr_cluster.go

```diff
@@ -256,7 +256,7 @@ func resourceAwsEMRClusterCreate(d *schema.ResourceData, meta interface{}) error
 	applications := d.Get("applications").(*schema.Set).List()
 
 	keepJobFlowAliveWhenNoSteps := true
-	if v, ok := d.GetOk("keep_job_flow_alive_when_no_steps"); ok {
+	if v, ok := d.GetOkExists("keep_job_flow_alive_when_no_steps"); ok {
```

Comment:
Is there a testcase that allows to check this? if not, could you write the smallest one so that we ensure there won't be regressions in the future?